examples/cactus-example-discounted-asset-trade/package.json

```diff
@@ -36,20 +36,25 @@
     "fabric-ca-client": "2.2.19",
     "fabric-network": "2.2.19",
     "http-errors": "1.6.3",
+    "indy-sdk": "1.16.0-dev-1655",
```

Comment:
indy-sdk was removed from package.json on purpose, it is described in this sample readme to install it manually. It was removed because it requries indy-sdk library to be installed on the system, which causes some troubles for CI and users.

Reply:
@outSH Ahhh, I remember now! I knew there was something special about that dependency! I think at that time we were still on Yarn v1 and it did not have this feature that solves this problem very easily in the root package.json by declaring that the native code of indy-sdk should not be built (that's how I got the CI to work in this PR).

This is not a perfect solution either though because people will still need to manually install the native dependencies themselves so we probably just went from confusing people in one way to confusing them in another. I'm not even sure which one is worse. 
If you think we should rather just not have the indy-sdk dependency declared in the package.json at all then I can update the PR with that, no problem, just let me know what you think in light of this new option we have in Yarn v3.

Reply:
@petermetz Please choose whichever you like, I'm removing indy SDK in a separate commit anyway :)

Reply:
@outSH Oh, fair enough haha! I'll just leave it as-is for now for the sake of simplicity (trying to save time, things are very busy!)